apps/examples/wifi_manager/wm_test/wm_test_stress_test2.c

```diff
@@ -30,6 +30,21 @@
 #define WM_TEST_TRIAL	   CONFIG_WIFIMANAGER_TEST_TRIAL
 #define WM_NSOFTAP_SSID "no_sta_0101" // for auto test
 #define TAG "[WTS2]"
+#define MAX_INTEROP_AP_CONFIG_COUNT 10
```

Comment:
you should make this parameter configurable by Kconfig. The number of testing AP can be changed. or implement it linked list.

Reply:
Sir, changes done as suggested. Please review.

Reply:
@pillip8282 When we change Kconfig, it needs change in defconfig to avoid CI build failure.
CI runs on artic, qemu also, do we need to change defconfig of other platforms also

Reply:
how about to add following code. 
'''
#ifndef CONFIG_WIFIMANAGER_MAX_... 
#define MAX_... default value
#else 
#define MAX_... CONFIG_WIFIMANAGER_MAX...
#endif
'''

Reply:
Sir, changes done as suggested. Please review.